activity_calendar/migrations/0005_auto_20201016_1445.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.15 on 2020-10-16 12:45
```

Comment:
Might as well squash these two migrations as they were introduced in the same branch and relate to the same model.